### DISCUSSION TITLE
Congratulations and questions

### DISCUSSION THREAD

**Discussion by frederikhors** [nxthat/nanocl#349] [Q&A] [answered]
I just found out `nanocl` and I'm still speechless. Thanks for your work!

Since I'm not very experienced I ask you:

- is it conceivable today to use `nanocl` instead of `kubernetes`? (I'm referring to a very simple use of `kubernetes`: nginx as a reverse-proxy and a few pods (deployments) of simple web applications);

- are there any dashboards that can be used (even very simple ones)?

- **are there online tutorials/guides?**

**Comment by 0xle0ne** [comment, marked as answer]
Hi @frederikhors, thanks for your interest in `nanocl`!

I've been using `nanocl` for production deployment of my services and find it to be a great option for personal dashboard/landing pages and services. However, if you require multi-node support, I would recommend using `kubernetes` instead. Unfortunately, `nanocl` doesn't provide this feature currently, and the API may be unstable - meaning the data structure and endpoints can change. In the context of high-availability production for a company, it's best to use `kubernetes` for now.

I am planning to develop a dashboard, but it's not yet available.
You can refer to our [installation guide](https://docs.next-hat.com/setups/nanocl/linux/ubuntu) for instructions on how to install `nanocl`, and our [get started](https://docs.next-hat.com/guides/nanocl/get-started/orientation-and-setup) guide to learn how to deploy simple web services.

Feel free to test and give us your feedback.
If you're interested, consider watching the repository for updates.
The stable version will be released as 1.0.0 with all features and should be available later this year.

**Comment by frederikhors** [comment]
I'll wait for the stable version.

Thanks!
